### DISCUSSION TITLE
Export template -> save to file

### DISCUSSION THREAD

**Discussion by toporivskiy** [lairdubois/lairdubois-opencutlist-sketchup-extension#430] [Ideas]
I really need to be able to create an export template and distribute it to people. To do this, I need to be able to **export and import the template to a file**.

Is there such a possibility? If not, will it?

**Comment by bbeaulant** [comment]
Hi,

Could you please explain what is an export template for you ? A specific layout ?

**Reply by toporivskiy** [reply]
I mean export to a table (csv).  In version 4.0, it became possible to customize this export, write formulas.  And there is a need to make a template with formulas and distribute it to people in the form of a file.  And for this I need the ability to save a template into a file, and import template from a file.

**Reply by bbeaulant** [reply]
Directly, you can't export templates to file, but an alternative exists !

Export templates are stored in the current **SKP file**. That mean that you can transform an empty file with configured export templates as a new SketchUp Model Type.
And then distribute it as a base of a new SketchUp document. 
And more, each user receiving this file could store the current export configuration to its **presets** and use it in any other SKP file.

**Reply by toporivskiy** [reply]
Thanks for the "lifehack". If it is possible to add such an export function to a file, it would be cool!

**Reply by bbeaulant** [reply]
It's more than just a hack. It is a desired solution to transfert all OpenCutList parameters in a file without revinvent the wheel :)
